### PR TITLE
all keywords added to lexical for scanner

### PR DESCRIPTION
**Pull request by Corcatraz**
All keywords are added but I'm just trying out this "Pull request".... the build was successful so I think its all good

### CODE REVIEW

File: .classpath [hunk 1]
```diff
@@ -1,28 +1,28 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (vthai) [inline .classpath]: will need to exclude this file from repo in the future, it just keep changing everytime but add no values, I will work on this one later

**Comment by vthai** [comment]
Green light from me :)

File: .classpath (hunk 1 above)
Comment (Corcatraz) [inline .classpath]: ah yea for sure. I need to remember how to add that to the gitIgnore, thats what I do yea?

Comment (vthai) [inline .classpath]: Yeah, that's right, but as it has already "lived" in the repo, so we can no "ignore" it with .gitignore, the proper step would be:

1) git rm --cached <filename>
2) commit it
3) add it to .gitignore

Comment (vthai) [inline .classpath]: or you could leave that for now, just merge to master, I will fix it on Master for the other 3 files: Lexer.java, sym.java as well as parser.java, these three files are guarantee to be generated by jflex and cup and they changes all the time, so no need to have them in repo as well

**Pull request merged**